
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>

  <title>Lu Cheng's Homepage</title>

  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="description" content="Lu Cheng is currently a Principal Investigator in Department of Computer Science, Aalto University.">
  <meta name="keywords" content="Lu Cheng, 程路, Bayesian statistics, Bioinformatics, Single cell, Metagenomics">
  <meta name="author" content="Lu Cheng" />

  <link rel="stylesheet" href="w3.css">

  <style>
  .w3-sidebar a {font-family: "Roboto", sans-serif}
  body,h1,h2,h3,h4,h5,h6,.w3-wide {font-family: "Montserrat", sans-serif;}
  </style>

  <!--
  <link rel="icon" type="image/png" href="images/icons.png">
  //-->

  <!--
  <script src="jquery.min.js"></script>
  <script>
  $(document).ready(function(){
    // Add smooth scrolling to all links
    $("a").on('click', function(event) {
      // Make sure this.hash has a value before overriding default behavior
      if (this.hash !== "") {
        // Prevent default anchor click behavior
        event.preventDefault();
        // Store hash
        var hash = this.hash;
        // Using jQuery's animate() method to add smooth page scroll
        // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
        $('html, body').animate({
          scrollTop: $(hash).offset().top
        }, 800, function(){
          // Add hash (#) to URL when done scrolling (default click behavior)
          window.location.hash = hash;
        });
      } // End if
    });
  });
  </script>
  //-->

</head>


<body class="w3-content" style="max-width:1000px">

<!-- Sidebar/menu -->
<nav class="w3-sidebar w3-bar-block w3-black w3-collapse w3-top w3-right" style="z-index:3;width:150px" id="mySidebar">
  <div class="w3-container w3-display-container w3-padding-16">
    <h3><b>LU CHENG</b></h3>
  </div>
  <div class="w3-padding-64 w3-text-light-grey w3-large" style="font-weight:bold">
    <a href="#home" class="w3-bar-item w3-button">Home</a>
    <a href="#news" class="w3-bar-item w3-button">News</a>
    <a href="#projects" class="w3-bar-item w3-button">Research</a>
    <!-- <a href="#talks" class="w3-bar-item w3-button">Talks</a> -->
    <a href="#publications" class="w3-bar-item w3-button">Publications</a>
    <a href="#service" class="w3-bar-item w3-button">Services</a>
    <!-- <a href="#award" class="w3-bar-item w3-button">Awards</a> -->
  </div>
</nav>

<!-- Top menu on small screens -->
<header class="w3-bar w3-top w3-hide-large w3-black w3-xlarge">
  <div class="w3-bar-item w3-padding-24">Lu Cheng</div>
  <a href="javascript:void(0)" class="w3-bar-item w3-button w3-padding-24 w3-right"  style="font-stretch: extra-expanded;" onclick="w3_open()"><b>≡</b></a>
  </div>
</header>

<!-- Overlay effect when opening sidebar on small screens -->
<div class="w3-overlay w3-hide-large" onclick="w3_close()" style="cursor:pointer" title="close side menu" id="myOverlay"></div>

<!-- !PAGE CONTENT! -->
<div class="w3-main" style="margin-left:150px">

  <!-- Push down content on small screens -->
  <div class="w3-hide-large" style="margin-top:83px"></div>

<!-- The Home Section -->
    <div class="w3-container w3-center w3-padding-32" id="home">
      <img style="width: 80%;max-width: 320px" alt="profile photo" src="images/lucheng.png">
      <h1>Lu Cheng</h1>
        <p class="w3-justify" style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;max-width:600px">
          I am the Principal Investigator (Academy Research Fellow) leading the <a href="">Bayesian Modelling for Biological Data</a> research group at the <a href="https://www.cs.aalto.fi/">Department of Computer Science</a>, Aalto University. My main research interests are developing statistical and machine learnig methods for applications in single cell biology, metagenomics, epigenetics.
        </p>

        <p class="w3-center">
          <a href="mailto:firstname.lastname@aalto.fi">Email</a> &nbsp/&nbsp
          <a href="https://scholar.google.fi/citations?user=Zy14QK0AAAAJ&hl=en">Google Scholar</a> &nbsp/&nbsp
          <a href="https://orcid.org/0000-0002-6391-2360">ORCID</a> &nbsp/&nbsp
        </p>
        </tbody></table>
  </div>

<!-- The News Section -->
  <div class="w3-container w3-light-grey w3-padding-32" id="news">
   <h2>News</h2>
   	  <p><li> 03/2022, our paper about estimating alternative polyadenylation (APA) from scRNA-seq has been accepted by <a href="https://doi.org/10.1093/nar/gkac167">Nucleic Acids Research</a>.</li></p>
      <p><li> 03/2022, our paper about oral microbiota in pregenant women is published in <a href="https://doi.org/10.1080/20002297.2022.2053389"> Journal of Oral Microbiology</a>.</li></p>
      <p><li> 01/2021, our paper discovering a deletion hotspot in Nsp1 gene of SARS-COV-2 genome is published in . <a href="https://doi.org/10.1016/j.chom.2021.01.015">Cell Host & Microbe</a>.</li></p>
      <!--
      <p><li> 01/2021, I will give a talk about AdderNet at <a href="https://haet2021.github.io/speakers.html">HAET ICLR 2021 workshop</a>.</li></p>
      <p><li> 12/2020, two papers have been accepted by <a href="https://aaai.org/Conferences/AAAI-21/">AAAI 2021</a>.</li></p>
      <p><li> 11/2020, I accepted the invitation to serve as an Area Chair for <a href="https://icml.cc/Conferences/2021">ICML 2021</a>.</li></p>
      <p><li> 09/2020, six papers have been accepted by <a href="https://nips.cc/Conferences/2020">NeurIPS 2020</a>.</li></p>
      <p><li> 07/2020, one paper has been accepted by <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=5962385">IEEE TNNLS</a>.</li></p>
      <p><li> 07/2020, one paper has been accepted by <a href="https://eccv2020.eu/accepted-papers/">ECCV 2020</a>.</li></p>
      <p><li> 06/2020, two papers have been accepted by <a href="https://icml.cc/Conferences/2020/AcceptedPapersInitial">ICML 2020</a>.</li></p>
      <p><li> 07/2020, one paper has been accepted by <a href="http://2020.acmmm.org/accepted-paper-id-list.txt">ACM MM 2020</a>.</li></p>
      <p><li> 02/2020, seven papers have been accepted by <a href="http://openaccess.thecvf.com/menu.py">CVPR 2020</a>.</li></p>
      <p><li> 01/2020, one paper has been accepted by <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=5962385">IEEE TNNLS</a>.</li></p>
      <p><li> 11/2019, three papers have been accepted by <a href="https://aaai.org/Conferences/AAAI-20/wp-content/uploads/2020/01/AAAI-20-Accepted-Paper-List.pdf">AAAI 2020</a>.</li></p>
      -->

  </div>
<!-- The Projects Section -->
  <div class="w3-container w3-padding-32" id="projects">
    <h2>Research Projects</h2>
    <p class="w3-justify">
        Biology has stepped into a new era where data are generated at an unprecedented pace. It is therefore necessary to develop new methods to analyse these ever increasing datasets. Thanks to the vast amount of data, researchers have a nice opportunity to come up with new biological hypotheses and look into the data from a new perspective. This kinds of projects may generate new statistical methods, new biological insights. In practice, we can make novel biological predictions and validate them in wet lab.  I believe this type of data-driven research paradigm will become more and more popular in the future.
    </p>

        <h4><li>Estimating RNA modifications using Nanopore data</li></h4>
        <img style="width:96%;" src="images/projects/nanopore.png"> 
        <p class="w3-justify">
        Oxford Nanopore (ONT) is the third generation sequencing technology. It measures the electric current when a RNA molecule passes through the pore. In principle, different signals are generated for different nucleotides. As we know, there exists different various modifications, e.g. m6A, in the naive RNA molecule. We aim to develop a Gaussain process based model to estimate these modifications from Nanopore data. 

        <h4><li>Estimating APA events from scRNA-seq data</li></h4>
        <img style="width:96%;" src="images/projects/scape.jpeg"> 
        <p class="w3-justify">
        <a style="color: #447ec9" href="https://github.com/LuChenLab/SCAPE">Project Page</a> | <a style="color: #447ec9" href="https://doi.org/10.1093/nar/gkac167">Paper</a>
        </p>
        <p class="w3-justify">
       Alternative polyadenylation (APA) is one kind of alternative splicing in the 3'-UTR part of a gene. Common single cell RNA-seq (scRNA-seq) technologies such as 10x happen to measure the 3'-UTR of the gene, thus scRNA-seq data could be used to estimate APA events. We notice that the insert size information is standard in the sequencing library prepartion. This important prior information help us to develop an elegant Bayesian mixture model that shows superior performance than state of the art. Next, we demonstrated differential APA usage at different levels including organs, cell types, cell states (tumour or normal). The top differentiated APA events also show a strong association with miRNA expression. Furthermore, we demonstrated genome-wide dynamic changes of APA usage during stem cell differentiation (erythropoiesis and iPSC).
        </p> 


        <h4><li>GWAS study of SARS-CoV-2 genomes</li></h4>
        <img style="width:96%;" src="images/projects/sars-cov-2.jpg"> 
        <p class="w3-justify">
        <a style="color: #447ec9" href="https://doi.org/10.1016/j.chom.2021.01.015">Paper</a>
        </p>
        <p class="w3-justify">
        We have got hundreds of SARS-COV-2 genomes from COVID-19 diagnosed patients and their clinical data from our collaborators. We ask what are the key genotypes of the SARS-COV-2 genome that affect the symptoms of the patients. Through a GWAS study, we find a deletion in the nsp1 gene that is associated with the severeness of the symptoms. This deletion location turns out to be a hopspot in many different SARS-COV-2 strains worldwide. Then we show that the deletion induces with lower type I interferon response in wet lab.  
        </p> 

        <h4><li>Additive Gaussian process model for longitudinal data</li></h4>
        <img style="width:96%;" src="images/projects/longp.png"> 
        <a style="color: #447ec9" href="https://github.com/chengl7/LonGP">Project Page</a> | <a style="color: #447ec9" href="https://doi.org/10.1038/s41467-019-09785-8">Paper</a>
        <p class="w3-justify">
        The study design of a biological experiment can be very complex, especially when collecting time series data from different individuals. Many different covariates in the longitudinal study design need to be considered. These covariates may affect the target variable in a nonlinear way. What are the important covariates and how do they affect the target variable? We developed a tool called LonGP to decompose the raw data into several linear/nonlinear functions of the covariates. Application of this method to a published longitudinal type 1 diabetes (T1D) proteomics dataset helps us to identify some proteins that show weak changes near the seroconversion event. 

        <!-- <img style="width:32%;" src="images/atlas200.png"> &nbsp&nbsp <img style="width:60%;" src="images/video.gif"> -->
  </div>
  
  <!-- comment start 
  <div class="w3-container w3-light-grey w3-padding-32" id="talks">
    <h2>Talks</h2>
      <p><li> 10/2021, Vision Transformer at <a href="https://haet2021.github.io/speakers.html">HAET ICLR 2021 workshop</a>.</li></p>
	  <p><li> 05/2021, Adder Neural Network at <a href="https://haet2021.github.io/speakers.html">HAET ICLR 2021 workshop</a>. Thanks <a href="https://datawisdom.ca/">Vahid</a> for the invitation.</li></p>
      <p><li> 06/2020, "<a href="http://valser.org/webinar/slide/slides/20200603/%E6%A8%A1%E5%9E%8B%E5%8E%8B%E7%BC%A9-%E5%B7%A5%E4%B8%9A%E7%95%8C%E5%92%8C%E5%AD%A6%E6%9C%AF%E7%95%8C%E7%9A%84%E5%B7%AE%E5%BC%82.pdf">AI on the Edge - Discussion on the Gap Between Industry and Academia</a>" at <a  href="http://valser.org/"><strong>VALSE</strong></a> Webinar.</li></p>
      <p><li> 05/2020, "<a href="https://www.bilibili.com/video/av925692420/">Edge AI: Progress and Future Directions</a>" at <a href="https://www.qbitai.com/"> <strong>QbitAI</strong></a> using <a  href="https://www.bilibili.com/"><strong>bilibili</strong></a>.</li></p>
  </div>

  comment end-->

 <!-- The Publications Section -->
  <div class="w3-container w3-padding-32" id="publications">
    <h2>Selected publications</h2>
      <p class="w3-left-align" style="line-height:200%">
        <span>&#10034;</span>: (co-)first author; #: (co-)coresponding author. For a full list, please check my <a href="https://scholar.google.fi/citations?user=Zy14QK0AAAAJ&hl=en">Google Scholar</a>. 
      </p>
    <ol>
     
     <!--

      <p>
      <li><strong></strong>
      <br>
      
      <br>
      <em></em> 2021 | <a style="color: #447ec9" href="">paper</a>
      </p>

    -->

      <p>
      <li><strong>SCAPE: a mixture model revealing single-cell polyadenylation diversity and cellular dynamics during cell differentiation and reprogramming</strong>
      <br>
      Ran Zhou*, Xia Xiao, Ping He, Yuancun Zhao, Mengying Xu, Xiuran Zheng, Ruirui Yang, Shasha Chen, Lifang Zhou, Dan Zhang, Qingxin Yang, Junwei Song, Chao Tang, Yiming Zhang, Jing-wen Lin<sup>#</sup>, <strong>Lu Cheng</strong><sup>#</sup>, Lu Chen<sup>#</sup>
      <br>
      <em>Nucleic Acids Research</em> 2022 | <a style="color: #447ec9" href="https://doi.org/10.1093/nar/gkac167">paper</a>
      </p>

      <p>
      <li><strong>Genomic monitoring of SARS-CoV-2 uncovers an Nsp1 deletion variant that modulates type I interferon response</strong>
      <br>
      Jing-wen Lin*<sup>#</sup> , Chao Tang* , Han-cheng Wei* , Baowen Du* , Chuan Chen* , Minjin Wang* , Yongzhao Zhou* , Ming-xia Yu* , <strong>Lu Cheng</strong>* , Suvi Kuivanen* , Natacha S. Ogando , Lev Levanov , Yuancun Zhao , Chang-ling Li , Ran Zhou , Zhidan Li , Yiming Zhang , Ke Sun , Chengdi Wang , Li Chen , Xia Xiao , Xiuran Zheng , Sha-sha Chen , Zhen Zhou , Ruirui Yang , Dan Zhang , Mengying Xu , Junwei Song , Danrui Wang , Yupeng Li , ShiKun Lei , Wanqin Zeng , Qingxin Yang , Ping He , Yaoyao Zhang , Lifang Zhou , Ling Cao , Feng Luo , Huayi Liu , Liping Wang , Fei Ye , Ming Zhang , Mengjiao Li , Wei Fan , Xinqiong Li , Kaiju Li , Bowen Ke , Jiannan Xu , Huiping Yang , Shusen He , Ming Pan , Yichen Yan , Yi Zha , Lingyu Jiang , Changxiu Yu , Yingfen Liu , Zhiyong Xu , Qingfeng Li , Yongmei Jiang , Jiufeng Sun , Wei Hong , Hongping Wei , Guangwen Lu , Olli Vapalahti , Yunzi Luo , Yuquan Wei , Thomas Connor , Wenjie Tan , Eric J. Snijder , Teemu Smura<sup>#</sup> , Weimin Li <sup>#</sup>, Jia Geng <sup>#</sup>, Binwu Ying <sup>#</sup>, Lu Chen<sup>#</sup>
      <br>
      <em>Cell Host & Microbe</em> 2021 | <a style="color: #447ec9" href="https://doi.org/10.1016/j.chom.2021.01.015">paper</a>
      </p>

      <p>
      <li><strong>An additive Gaussian process regression model for interpretable non-parametric analysis of longitudinal data</strong>
      <br>
      <strong>Lu Cheng</strong>*<sup>#</sup>, Siddharth Ramchandran, Tommi Vatanen, Niina Lietzén, Riitta Lahesmaa, Aki Vehtari & Harri Lähdesmäki<sup>#</sup>
      <br>
      <em>Nature Communications</em> 2019 | <a style="color: #447ec9" href="https://doi.org/10.1038/s41467-019-09785-8">paper</a>
      </p>

      <p>
      <li><strong>Hierarchical and spatially explicit clustering of DNA sequences with BAPS software</strong>
      <br>
      <strong>Lu Cheng</strong>*, Thomas R. Connor, Jukka Sirén, David M. Aanensen, Jukka Corander<sup>#</sup>
      <br>
      <em>Molecular Biology and Evolution</em> 2013 | <a style="color: #447ec9" href="">paper</a>
      </p>

      <p>
      <li><strong>Bayesian estimation of bacterial community composition from 454 sequencing data</strong>
      <br>
      <strong>Lu Cheng</strong>*<sup>#</sup>, Alan W. Walker, Jukka Corander
      <br>
      <em>Nucleic Acids Research</em> 2012 | <a style="color: #447ec9" href="https://doi.org/10.1093/nar/gks227">paper</a>
      </p>



      </ol>

    </p>
  </div>

<!-- The Services Section -->
  <div class="w3-container w3-light-grey w3-padding-32" id="service">
    <h2>Services</h2>
      <p><li> Reviewers of NeurIPS 2016, 2017, 2018, 2019, 2021, 2022.</p>
      <p><li> Reviewers of ICLR 2018, 2021, 2022.</p>
      <p><li> Reviewers of Nucleic Acid Research, Neurocomputing, Ecology Evolution, Genome Biology Evolution, Plos Computational Biology, Plos One, BMC Bioinformatics.</p>
  </div>

  <!-- The Awards Section 
  <div class="w3-container w3-padding-32" id="award">
    <h2>Awards</h2>
    <p><li> 2020, <a href="https://mp.weixin.qq.com/s/dORL01lgFNDHgjp3KMJmiQ">Nomination for Outstanding Youth Paper Award</a>, <a href="https://worldaic.com.cn/portal/en/aboutus.html">WAIC</a></p>               
    <p><li> 2017, <a href="https://research.google/outreach/phd-fellowship/recipients/?category=2017">Google PhD Fellowship</a></p>
    <p><li> 2017, <a href="http://scholarship.baidu.com/">Baidu Scholarship</a></p>
    <p><li> 2017, President's PhD Scholarship, Peking University</p>
    <p><li> 2017, National Scholarship for Graduate Students</p>
    <p><li> 2016, National Scholarship for Graduate Students</p>
  </div>  

  -->


  <div class="w3-light-grey w3-center w3-padding-24">

    Welcome to use this website's <a href="https://github.com/YunheWang/HomePage">source code</a>, just add a link back to here. <a href="https://www.wangyunhe.site/">&#10025;</a></br>

  <!-- Default Statcounter code for Yunhe Wang's Homepage
  https://www.wangyunhe.site -->
  No.
  <script type="text/javascript">
  var sc_project=12347113; 
  var sc_invisible=0; 
  var sc_security="21aca5d1"; 
  var sc_https=1; 
  var scJsHost = "https://";
  document.write("<sc"+"ript type='text/javascript' src='" + scJsHost+
  "statcounter.com/counter/counter.js'></"+"script>");
  </script> Visitor Since April 2022. Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-opacity">w3.css</a>
  <noscript>
    <div class="statcounter"><a title="Web Analytics Made Easy -
  StatCounter" href="https://statcounter.com/" target="_blank"><img
  class="statcounter" src="https://c.statcounter.com/12347113/0/21aca5d1/0/"
  alt="Web Analytics Made Easy - StatCounter"></a></div>
  </noscript>
  <!-- End of Statcounter Code -->

  </div>

  <!-- End page content -->
</div>

<script>
// Accordion 
function myAccFunc() {
  var x = document.getElementById("demoAcc");
  if (x.className.indexOf("w3-show") == -1) {
    x.className += " w3-show";
  } else {
    x.className = x.className.replace(" w3-show", "");
  }
}

// Click on the "Jeans" link on page load to open the accordion for demo purposes
document.getElementById("myBtn").click();


// Open and close sidebar
function w3_open() {
  document.getElementById("mySidebar").style.display = "block";
  document.getElementById("myOverlay").style.display = "block";
}
 
function w3_close() {
  document.getElementById("mySidebar").style.display = "none";
  document.getElementById("myOverlay").style.display = "none";
}
</script>

</body>
</html>
